libs/blocks/comparison-table/comparison-table.css

```diff
@@ -0,0 +1,490 @@
+:root {
+  --comparison-border-radius: 8px;
+  --comparison-border-color: #d4d4d4;
```

Comment:
It's advisable to use the global variables where possible. This, for example, is already defined as `--color-gray-300`. `styles.css` will surely contain some more of the values defined here.